api/src/main/resources/liquibase/3.x-liquibase-changeSet.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+-->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
So this all looks correct, but I'm a bit concerned about having this permanently in the module, mostly because it means we're evaluating this liquibase changeset each time things start-up. Do you think it makes sense to apply these changes to, say, AMPATH's database and instead revise the existing changesets to produce the tables we now expect (with a couple of `validCheckSum` for the current checksums from the file to ensure those changes aren't re-applied to AMPATH)?

Reply:
Make sense! I can delete the tables/drop columns manually, then remove respective changesets. I don't think we need to add `validCheckSum` because will be deleting the entire changeset.